invenio_app_ils/circulation/views.py

```diff
@@ -33,6 +30,7 @@ def decorator_builder(f):
         @wraps(f)
         def decorate(*args, **kwargs):
             check_permission(
+                action,
```

Comment:
The action should not be exposed and since its used to get the permission, we don't need to pass it to `check_permission` as argument.

Reply:
@equadon is there any reason you pass the action explicitly?

Reply:
@zzacharo We passed the action along with the permission to the permission error class before but it was changed. It's now removed.